README.md

```diff
@@ -54,7 +54,7 @@ npm install paginated-connection
 Here is a basic example to get you started with Paginated Connection:
 
 ```typescript
-import { paginatedConnection, PaginationInput } from 'paginated-connection';
+const { paginatedConnection, PaginationInput } = require('@treedom/paginated-connection')
```

Comment:
this is a typescript file example, probably should be import syntax instead of require

Reply:
Oh yes you are right, i've just used in a javascript project. 

Let me change